seqr/views/apis/summary_data_api.py

```diff
@@ -348,3 +349,122 @@ def _get_airtable_collaborator_names(user, collaborator_ids):
         collaborator_id: collaborator_map.get(collaborator_id, {}).get('CollaboratorID')
         for collaborator_id in collaborator_ids
     }
+
+
+@login_and_policies_required
+def family_metadata(request, project_guid):
+    projects, _ = _get_metadata_projects(request, project_guid)
```

Comment:
copy-pasted from report_api, this is the only line that changed (uses a different helper function to get the allowed projects)